RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 4.1
 -----
 - Fix an intermittent crash when downloading Orders
+- The Photo Library permission alert shouldn't be prompted except for when uploading images in debug builds with Products M2 enabled.
```

Comment:
I think this should be labeled `[internal]`

Reply:
Since the use `DefaultProductUIImageLoader` isn't behind a feature flag, this is actual an issue for the current release 😓 (though, the alert seems to be triggered only for below iOS 13 from my testing, and that's a small percentage of users. I will add a note about how it's reproducible only in iOS 11/12!)